cmds/core/stat/stat_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2025 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build darwin || (linux && !arm && !386 && !mips && !mipsle)
+
+package main
+
+import (
+	"bytes"
+	"io"
+	"strings"
+	"testing"
+)
+
+func TestRun(t *testing.T) {
+	t.Run("no error", func(t *testing.T) {
+		errCode := run(io.Discard, io.Discard, "./stat.go")
+		if errCode != 0 {
+			t.Errorf("expected 0 got %d", errCode)
+		}
+	})
+
+	t.Run("file not exists", func(t *testing.T) {
```

Comment:
Maybe we should also add a test that verifies each attribute for one, well-known file that we create once, such as file size, gid, last edited etc?

Reply:
Sure most of attributes can be verified in such way, but some of them are problematic so I decide to omit them for now.